### PR TITLE
Proof fixes

### PR DESCRIPTION
**Pull request by AhmedKorim**

### CODE REVIEW

File: examples/sdk-mixer/src/mixer.worker.ts [hunk 1]
```diff
@@ -1,3 +1,3 @@
-import { ProvingManagerWrapper } from '@webb-tools/sdk-core';
+import { ProvingMangerWrapper } from '@webb-tools/sdk-core';
```
Comment (drewstone) [inline examples/sdk-mixer/src/mixer.worker.ts:1]: Spelling is off

File: examples/sdk-mixer/src/mixer.worker.ts [hunk 2]
```diff
@@ -1,3 +1,3 @@
...
 
-new ProvingManagerWrapper();
+new ProvingMangerWrapper();
```
Comment (drewstone) [inline examples/sdk-mixer/src/mixer.worker.ts:3]: Spelling is off

**Review by drewstone — CHANGES_REQUESTED** [review]

File: examples/sdk-mixer/src/mixer.worker.ts (hunk 2 above)
Comment (AhmedKorim) [reply examples/sdk-mixer/src/mixer.worker.ts:3]: this is fixed already, right?

Comment (AhmedKorim) [reply examples/sdk-mixer/src/mixer.worker.ts:3]: this is fixed already, right?

**Pull request merged**